kolibri/plugins/coach/assets/src/modules/classSummary/index.js

```diff
@@ -469,6 +470,16 @@ export default {
             store.dispatch('fetchLessonsSizes', classId),
             store.dispatch('fetchQuizzesSizes', classId),
           ]).then(() => summary);
+        })
+        .catch(error => {
+          if (error.response.status === 403) {
+            // Loading the class summary is somewhat of an entry point - if this 403s
+            // we can reasonably redirect the user to the Kolibri root URL. This was
+            // done to gracefully handle the case that an admin is signed out due to
+            // timeout while in Coach and when a Learner signs in, they are returned
+            // to the URL the admin was previously on and hit this 403
+            redirectBrowser();
```

Comment:
This is not fixing the issue as described - the mismatch in the behaviour was specifically in the quiz creation page, where it is not using the authentication wrapper component to show meaningful errors to the user in the case that they are not authenticted.

Also, as this is now applying to every page in coach, the existing authentication wrapper errors may not show because of this.

Further, if a coach's session expired, and the classSummary API update call returned early, they would now get redirected to the login page, but without a `next` parameter to guide them back to where they were before.